src/containers/pages/InterventionPlan/IRS/index.tsx

```diff
@@ -240,7 +240,8 @@ interface DispatchedStateProps {
 }
 
 const mapStateToProps = (state: Partial<Store>, ownProps: any): DispatchedStateProps => {
-  const isDraftsList = ownProps.path === INTERVENTION_IRS_DRAFTS_URL;
+  const isDraftsList =
+    ownProps.path === INTERVENTION_IRS_DRAFTS_URL || ownProps.path === INTERVENTION_IRS_URL;
```

Comment:
Initial bug was caused by this conditional failing to recognize `/intervention/irs` as an IRS path that should render draft plans, so [the container didn't fetch plans from OpenSRP](https://github.com/onaio/reveal-frontend/blob/master/src/containers/pages/InterventionPlan/IRS/index.tsx#L111-L129).

Reply:
An alternate fix would simply be removing the `/intervention/irs` from the App and updating the IRS link on the Home page to point to `/intervention/irs/drafts`.

Reply:
@moshthepitt as for tests it looks like they're currently pretty limited, I imagine we could count the number of plans returned in the props and simply mount the component with different routes (relates to https://github.com/onaio/reveal-frontend/issues/669).

Reply:
@cKellyDesign for a test is it not possible to recreate what happened (to cause the issue) and confirm that now we get a different result?  It just feels weird to me that we are adding a fix that could later be removed accidentally or otherwise and we wouldn't know until "they" tell us oh no this bug is back magically.

Reply:
@moshthepitt sorry, I mean we can count rows in both draft cases, I think to achieve the different cases we can mount `App` and push the history to navigate to all the URLs for which this container is rendered.

Reply:
> sorry, I mean we can count rows in both draft cases, I think to achieve the different cases we can mount App and push the history to navigate to all the URLs for which this container is rendered.

🙈 not clear what you mean, but I'll get the idea when we have some code.

Reply:
🙊 pole, yes I think it will be more clear when I commit some code :)